src/cephadm/cephadm

```diff
@@ -2491,6 +2491,12 @@ def command_check_host():
     if not check_time_sync():
         raise Error('No time synchronization is active')
 
+    if args.expect_hostname:
+        if get_hostname() != args.expect_hostname:
+            raise Error('hostname "%s" does not match expected hostname "%s"' % (
+                get_hostname(), args.expect_hostname))
+        logger.info('Hostname "%s" matches expected', args.expect_hostname)
```

Comment:
The wording of this log line somehow feels awkward .. is it needed? maybe just remove?